rosbag2_transport/src/rosbag2_transport/recorder.cpp

```diff
@@ -46,6 +46,8 @@
 
 namespace
 {
+// TODO(emersonknapp) re-enable subscription_qos_for_topic once the cyclone situation is resolved
+#ifdef ROSBAG2_ENABLE_ADAPTIVE_QOS_SUBSCRIPTION
```

Comment:
I had to introduce this `ifdef` since I was getting an unused function compile time error.